### PR TITLE
Update it.po

### PR DESCRIPTION
**Pull request by albanobattistella**

### CODE REVIEW

**Comment by albanobattistella** [comment]
Hi,

I fixed but checks failed?

File: po/it.po [hunk 1]
```diff
@@ -174,18 +175,26 @@ msgid ""
 "your head and onto your to-do list. From there you can begin to organize and "
 "prioritize so you know exactly what to focus on and when."
 msgstr ""
+"Si scopre che i nostri cervelli sono in realtà cablati per farci pensare ai nostri "
+"compiti incompiuti. Pratico quando hai una cosa su cui devi lavorare. Non così"
+"bene quando hai più di 30 compiti in lizza per la tua attenzione contemporaneamente. Ecco perché"
+"il primo passo per organizzare il tuo lavoro e la tua vita è ottenere tutto dalla"
+"tua testa e il tuo elenco di cose da fare. Da lì puoi iniziare a organizzare la"
+"priorità in modo da sapere esattamente su cosa si deve concentrarsi e quando.
```
Comment (alainm23) [inline po/it.po]: The string has not been closed `"`

File: po/it.po [hunk 2]
```diff
@@ -217,16 +229,23 @@ msgid ""
 "- (Optional) Select a different project color from the color list.\n"
 "- Click Add to create the project."
 msgstr ""
+"- Sia che tu stia pianificando una presentazione, preparando un evento o creando"
+"un sito Web, crea un progetto in modo che tutti i dettagli importanti vengano salvati in un"
+"posto centrale.\n"
+"- Nel menu di navigazione a sinistra, in basso, fai clic sul simbolo +.\n"
+"- Nel menu delle opzioni seleziona" Progetto "e digita il nome del tuo nuovo"
```
Comment (alainm23) [inline po/it.po]: A syntax error.

File: po/it.po [hunk 3]
```diff
@@ -381,25 +412,30 @@ msgid ""
 "Plan your week ahead with the Upcoming view. It shows everything on your "
 "agenda for the coming days: scheduled to-dos and calendar events."
 msgstr ""
+"Pianifica la tua settimana in anticipo con la vista Imminente. Mostra tutto sulla tua"
+"agenda per i prossimi giorni: cose da fare e eventi del calendario programmati".
```
Comment (alainm23) [inline po/it.po]: syntax error

File: po/it.po [hunk 4]
```diff
@@ -869,31 +901,31 @@ msgstr "Crea"
...
+msgstr "Sei sicuro di voler resettare tutto?"
 
 #: src/Dialogs/Preferences.vala:458
 msgid ""
 "It process removes all stored information without the possibility of undoing "
-"it."
+"it.""
 msgstr ""
+"Questo processo rimuove tutte le informazioni memorizzate senza la possibilità di
+"annullarlo.""
```
Comment (alainm23) [inline po/it.po]: Two quotes are invalid.

File: po/it.po [hunk 5]
```diff
@@ -909,30 +941,33 @@ msgid ""
 "All the shortcuts to save you time! Some can be used anywhere in the app, "
 "while others only work when adding or editing tasks."
 msgstr ""
+"Tutte le scorciatoie per farti risparmiare tempo! Alcune possono essere utilizzate ovunque nell'app",
```
Comment (alainm23) [inline po/it.po]: syntax error

File: po/it.po [hunk 6]
```diff
@@ -997,61 +1033,65 @@ msgstr ""
...
+msgstr "📅️ Visualizza i tuoi eventi e pianifica meglio la tua giornata."
 
 #: data/com.github.alainm23.planner.appdata.xml.in:20
 #: data/com.github.alainm23.planner.appdata.xml.in:80
 msgid ""
 "💎️ Magic Button arrived: Drag and Drop and create your tasks wherever you "
 "want."
 msgstr ""
+"💎️ Pulsante magico arrivato: trascina la selezione e crea le tue attività ovunque tu"
+"voglia".
```
Comment (alainm23) [inline po/it.po]: Delete the point

File: po/it.po [hunk 7]
```diff
@@ -1098,121 +1138,131 @@ msgstr ""
...
 #: data/com.github.alainm23.planner.appdata.xml.in:56
 msgid "A financing page was added in the preferences."
-msgstr ""
+msgstr "Una pagina di finanziamento è stata aggiunta nelle preferenze."
 
 #: data/com.github.alainm23.planner.appdata.xml.in:57
 msgid ""
 "Planner menus have been redesigned. Making it more beautiful and productive."
 msgstr ""
+"I menu del pianificatore sono stati riprogettati. Rendendolo più bello e produttivo.
```
Comment (alainm23) [inline po/it.po]: end-of-line within string

File: po/it.po [hunk 8]
```diff
@@ -1098,121 +1138,131 @@ msgstr ""
...
 #: data/com.github.alainm23.planner.appdata.xml.in:64
 msgid "[GitHub #136] - Add new task button disappears."
-msgstr ""
+msgstr "[GitHub # 136] - Il pulsante aggiungi nuova attività scompare."
 
 #: data/com.github.alainm23.planner.appdata.xml.in:65
 msgid ""
 "[GitHub #145, #146] - The way to create the tutorial project was improved."
 msgstr ""
+"[GitHub # 145, # 146] - È stato migliorato il modo di creare il progetto tutorial.
```
Comment (alainm23) [inline po/it.po]: end-of-line within string

File: po/it.po [hunk 9]
```diff
@@ -199,11 +208,14 @@ msgid ""
 "- If you want to delete the due date, repeat the process and select the "
 "\"undate\" option."
 msgstr ""
+"- Se sai che devi eseguire l'attività in un determinato giorno, fai clic su"
+"icona del calendario e seleziona una data.\n"
+"- Se desideri eliminare la data di scadenza, ripeti la procedura e seleziona"
+"Opzione "\"senza data\."
```
Comment (alainm23) [inline po/it.po]: syntax error

**Comment by alainm23** [comment]
Thank you very much for your translation. I'm sorry I couldn't review this earlier. There are some syntax errors, I would recommend that you use [Poedit](https://poedit.net/) so that you can worry only about the translation.

**Review by alainm23 — CHANGES_REQUESTED** [review]
Thank you very much for your translation. I'm sorry I couldn't review this earlier. There are some syntax errors, I would recommend that you use [Poedit](https://poedit.net/) so that you can worry only about the translation.

**Comment by alainm23** [comment]
Could you send in a new PR please? I've updated the tradition files.

**Comment by albanobattistella** [comment]
Ok...Will be done!
________________________________
Da: Alain <notifications@github.com>
Inviato: lunedì 3 febbraio 2020 12:12:31
A: alainm23/planner <planner@noreply.github.com>
Cc: albanobattistella <albano_battistella@hotmail.com>; Author <author@noreply.github.com>
Oggetto: Re: [alainm23/planner] Update it.po (#193)


Could you send in a new PR please? I've updated the tradition files.

**Comment by albanobattistella** [comment]
done!

**Comment by alainm23** [comment]
Thanks :)

**Pull request merged**